lib/experimental/fonts/font-library/class-wp-rest-font-faces-controller.php

```diff
@@ -255,6 +347,30 @@ public function delete_item( $request ) {
 		return parent::delete_item( $request );
 	}
 
+	/**
+	 * Prepares a single post output for response.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @param WP_Post         $item    Post object.
+	 * @param WP_REST_Request $request Request object.
+	 * @return WP_REST_Response Response object.
+	 */
+	public function prepare_item_for_response( $item, $request ) {
+		$data = array();
+
+		$data['id']                 = $item->ID;
+		$data['theme_json_version'] = 2;
```

Comment:
```suggestion
		$data['theme_json_version'] = WP_Theme_JSON::LATEST_SCHEMA;
```

Reply:
I've changed check_font_face_data to use  `WP_Theme_JSON::LATEST_SCHEMA` when checking the response data, so tests will fail if the theme.json version increases.